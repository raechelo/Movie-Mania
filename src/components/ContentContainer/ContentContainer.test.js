import React from 'react';
import { shallow } from 'enzyme';

describe('ContentContainer', () => {

  let wrapper;

  beforeEach(() => {
    wrapper = shallow( <ContentContainer /> )
  });

  it('should match the snapshot with all the data passed in', () => {
    expect(wrapper).toMatchSnapshot()
  })
})