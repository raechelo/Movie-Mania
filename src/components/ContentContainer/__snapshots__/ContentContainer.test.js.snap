// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContentContainer should match the snapshot with all the data passed in 1`] = `
<section
  className="Content-Container"
>
  <Route
    component={
      Object {
        "$$typeof": Symbol(react.memo),
        "WrappedComponent": [Function],
        "compare": null,
        "displayName": "Connect(Home)",
        "type": [Function],
      }
    }
    exact={true}
    path="/"
  />
  <Route
    component={[Function]}
    exact={true}
    path="/favorites"
  />
  <Route
    component={[Function]}
    exact={true}
    path="/login"
  />
  <Route
    component={
      Object {
        "$$typeof": Symbol(react.memo),
        "WrappedComponent": [Function],
        "compare": null,
        "displayName": "Connect(SignUp)",
        "type": [Function],
      }
    }
    exact={true}
    path="/sign-up"
  />
</section>
`;
